frontend/src/components/roadmap/sections/RoadmapWhatWereWorkingOn.tsx

```diff
@@ -0,0 +1,50 @@
+import { useMessages, useTranslations } from "next-intl";
+
+import GithubIssueLink, { gitHubLinkForIssue } from "src/components/GithubLink";
+import RoadmapPageSection from "src/components/roadmap/RoadmapPageSection";
+
+export default function RoadmapWhatWereWorkingOn() {
+  const t = useTranslations("Roadmap.sections.progress");
+  const messages = useMessages() as unknown as IntlMessages;
+  const { contentItems } = messages.Roadmap.sections.progress;
+
+  return (
+    <RoadmapPageSection
+      sectionTitle={t("title")}
+      extraClasses="bg-white"
+      sectionContent={
+        <>
+          <div className="margin-top-1" />
+          {contentItems.map((i, j) => (
```

Comment:
for consistency, and other reasons that I probably have unreasonably strong opinions about, I'd rather stay away from abbreviated variable names like this. Can we generally use names that points to what the variables represent instead?